src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```
@@ -125,7 +125,7 @@ err.runtime.link.jdk.jlink.prohibited=This JDK does not contain packaged modules
 err.runtime.link.packaged.mods=This JDK has no packaged modules.\
 \ --keep-packaged-modules is not supported
 err.runtime.link.modified.file={0} has been modified
-err.runtime.link.patched.module=File {0} not found in the modules image.\
+err.runtime.link.patched.module=The current runtime has been patched with --patch-module.\
 \ --patch-module is not supported when linking from the run-time image
```

```suggestion
err.runtime.link.patched.module=jlink does not support linking from the run-time image when running on a patched runtime with --patch-module
```

This error message seems more direct.

---

OK, thanks!